charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml

```diff
@@ -159,7 +159,7 @@ data:
                         "dashes": false,
                         "datasource": "$datasource",
                         "decimals": 3,
-                        "description": "How much error budget is left looking at our 0.990% availability gurantees?",
+                        "description": "How much error budget is left looking at our 0.990% availability guarantees?",
```

Comment:
Please see the note at the top of this file.

Reply:
Oh, let me create a PR upstream and remove this. Thanks.

Reply:
https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/513

Reply:
Removed.